### DISCUSSION TITLE
removing underline in a textrun

### DISCUSSION THREAD

**Discussion by gjblajian** [dolanmiu/docx#1807] [Q&A] [answered]
@dolanmiu 
Let's say you have a style.id that defines underline for a heading/title and you want to override that underline at the textrun level (un-underline). How would you accomplish this? For example the paragraph is marked with a style that underlines the entire paragraph but you have 2 textRuns that are NOT to be underlined. 

There does not appear to be an {underline: {type: NONE}}
For bold and italic you can just set them as false.

**Comment by dolanmiu** [comment, marked as answer]
A fix will be added here:

https://github.com/dolanmiu/docx/pull/1822

This will add the `NONE` property to `UnderlineType`

Example of usage can be seen here:

https://github.com/dolanmiu/docx/blob/master/demo/2-declaritive-styles.ts
